chat-components/src/components/prechatsurveypane/common/defaultProps/defaultStyles/defaultPreChatSurveyPaneButtonStyles.ts

```diff
@@ -3,5 +3,7 @@ import { IPreChatSurveyPaneButtonStyles } from "../../../interfaces/IPreChatSurv
 export const defaultPreChatSurveyPaneButtonStyles: IPreChatSurveyPaneButtonStyles = {
     backgroundColor: "rgb(49, 95, 162)",
     color: "#FFFFFF",
-    fontFamily: "Segoe UI, Arial, sans-serif"
+    fontFamily: "Segoe UI, Arial, sans-serif",
```

Comment:
Is this font-family referred on all the devices ? If so, we might need to think of font support for iOS devices
E.g., something like this
-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif;

Reply:
This font family is used in other components as well. I will discuss this with the team

Reply:
Added a User Story for this: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3114058